cfgov/paying_for_college/models/disclosures.py

```diff
@@ -389,6 +390,22 @@ def as_json(self):
     def __str__(self):
         return self.primary_alias + " ({})".format(self.school_id)
 
+    @property
+    def program_codes(self):
```

Comment:
Could we have some tests for this method? I suspect coverage is passing because it's accessed in a call to `as_json` below, but I just want to make sure it's covered so that we don't break it sometime in the future.

Reply:
√

Reply:
@willbarton Test added.